tests/test_completion.py

```diff
@@ -78,7 +78,7 @@ def verify(self, *, completion_items: List[Dict[str, Any]], insert_text: str, ex
     def test_none(self) -> 'Generator':
         self.set_response("textDocument/completion", None)
         self.view.run_command('auto_complete')
-        yield lambda: self.view.is_auto_complete_visible()
+        yield lambda: self.view.is_auto_complete_visible() is False
```

Comment:
Very odd :) why is this test supposed to be inverted?

Reply:
that test was failing:
```
ERROR: test_none (test_completion.QueryCompletionsTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/predrag/.config/sublime-text/Packages/UnitTesting/unittesting/core/py33/case.py", line 28, in _executeTestPart
    yield from deferred
  File "/home/predrag/.config/sublime-text/Packages/LSP/tests/test_completion.py", line 81, in test_none
    yield lambda: self.view.is_auto_complete_visible()
TimeoutError: Condition not fulfilled within 4.00 seconds
```

And than I thought about that, and it makes sense that the AC is not visible (is False) if we set an empty list: 
```
completion_list.set_completions([])
completion_list.set_completions(None)
```

In both cases I don't expect to see the AC.